dashboard/app/models/paired_user_level.rb

```diff
@@ -0,0 +1,20 @@
+# == Schema Information
+#
+# Table name: paired_user_levels
+#
+#  id                      :integer          not null, primary key
+#  driver_user_level_id    :integer
+#  navigator_user_level_id :integer
+#  created_at              :datetime         not null
+#  updated_at              :datetime         not null
```

Comment:
Should we store `driver_user_id` and `navigator_user_id`?

If we don't, it will be inefficient to determine (for example) how much of a given classroom used pair programming.